data.sql

```diff
@@ -5,3 +5,14 @@ VALUES ('Agumon', '2020-02-03', 0, TRUE, 10.23),
        ('Gabumon', '2018-11-15', 2, TRUE, 8),
        ('Pikachu', '2021-01-07', 1, FALSE, 15.04),
        ('Devimon', '2017-05-12', 5, TRUE, 11);
+
+/* Add more data to animals table */
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Charmander', '2020-02-08', 0, FALSE, -11),
```

Comment:
- Hey, the data type for the weight is a floating number(decimal). let's ensure we've passed the correct data type to this column `'-11.0'`